lib/ipmi_sol.c

```diff
@@ -648,10 +648,12 @@ ipmi_print_sol_info(struct ipmi_intf * intf, uint8_t channel)
  *
  * function will return -1 if value is not valid, or
  * will return 0 if valid.
+ *
+ * base parameter currently unused, left in for extension.
  */
 int ipmi_sol_set_param_isvalid_uint8_t( const char *strval,
 					const char *name,
-					int base,
+					int __UNUSED__(base),
```

Comment:
Why did you decide to leave it? It's apparently a left-over from incomplete commit 94c0bd0a that replaced `strtol()` with `str2uint()` but forgot to remove the unused argument.